index.html

```diff
@@ -15,176 +15,247 @@
-                    <button type="button" class="btn1">See Project</button>
+            </div>
+            <div class="dash">
+                <div class="dash-mob">
+                   <h2 class="mob-tit">Profesional Art Printing Data</h2> 
+                   <p class="mob-dics">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                   <ul class="mob-li">
+                       <li>HTML</li>
+                       <li>BootStrap</li>
+                       <li>Rubby</li>
+                   </ul>
+                   <button type="button">See Project</button>
                 </div>
-                <div class="btn-list">
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard</p>
-                        <ul>
-                            <li class="list-btn">html</li>
-                            <li class="list-boot">bootstrap</li>
-                            <li class="list-btn">Ruby</li>
-                        </ul>
-                    <button type="button" class="btn1">See Project</button>
+                <div class="dash-mob">
+                    <h2 class="mob-tit">Profesional Art Printing Data</h2> 
+                    <p class="mob-dics">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="mob-li">
+                        <li>HTML</li>
+                        <li>BootStrap</li>
+                        <li>Rubby</li>
+                    </ul>
+                    <button type="button">See Project</button>
                 </div>
-                <div class="btn-list">
-                    <h3>Profesional Art Printing Data</h3>
-                    <p>A daily selection of privately personalized reads; no accounts
-                        or sign-ups required. has been the industry's standard</p>
-                        <ul>
-                            <li class="list-btn">html</li>
-                            <li class="list-boot">bootstrap</li>
-                            <li class="list-btn">Ruby</li>
-                        </ul>
-                    <button type="button" class="btn1">See Project</button>
+                <div class="dash-mob">
+                    <h2 class="mob-tit">Profesional Art Printing Data</h2> 
+                    <p class="mob-dics">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul class="mob-li">
+                        <li>HTML</li>
+                        <li>BootStrap</li>
+                        <li>Rubby</li>
+                    </ul>
+                    <button type="button">See Project</button>
                 </div>
```

Comment:
- [ ] There's a See Project button that takes you to the Live Demo of the featured project or to the GitHub repository. So kindly implement it.